src/quo2/components/wallet/account_overview/component_spec.cljs

```diff
@@ -0,0 +1,66 @@
+(ns quo2.components.wallet.account-overview.component-spec
+  (:require [quo2.components.wallet.account-overview.view :as account-overview]
+            [test-helpers.component :as h]))
+
+(h/describe
+  "Account overview test"
+  (h/test "renders correct account name"
+    (h/render [account-overview/view
+               {:account-name "Diamond Hand"}])
+    (h/is-truthy (h/get-by-text "Diamond Hand"))
+
+    (h/test "renders correct account value"
+      (h/render [account-overview/view
+                 {:current-value "20.0$"}])
+      (h/is-truthy (h/get-by-text "20.0$")))
+
+    (h/test "renders correct account changes"
+      (h/render [account-overview/view
+                 {:currency-change   "€0.00"
+                  :percentage-change "0.00%"}])
+      (and
+       (h/is-truthy (h/get-by-text "0.00%"))
+       (h/is-truthy (h/get-by-text "€0.00"))))
+
+    (h/test "renders correct timeframe"
+      (h/render [account-overview/view
+                 {:time-frame-string    "15 May"
+                  :time-frame-to-string "19 May"}])
+      (and
```

Comment:
`and` shouldn't be used because the assertion can either fail or pass. `h/is-truthy` is an assertion. If they fail they short-circuit the test.

Reply:
Where I used `and` in the test, _always_ showed together in their UI.

Reply:
I didn't quite understand your last message @ibrkhalil.

What I mean is that `h/is-truthy` is a final assertion, there's no meaningful return value for component specs, so `and` is doing nothing here. The function is named `is-truthy` to resemble the `cljs.test/is` macro`, which fails the test on falsey values.